spki/src/lib.rs

```diff
@@ -21,14 +21,15 @@
 //! Borrow the [`ObjectIdentifier`] first then use [`der::AnyRef::from`] or `.into()`:
 //!
 //! ```
+//! use der::asn1::Any;
 //! use spki::{AlgorithmIdentifier, ObjectIdentifier};
 //!
 //! let alg_oid = "1.2.840.10045.2.1".parse::<ObjectIdentifier>().unwrap();
 //! let params_oid = "1.2.840.10045.3.1.7".parse::<ObjectIdentifier>().unwrap();
 //!
 //! let alg_id = AlgorithmIdentifier {
 //!     oid: alg_oid,
-//!     parameters: Some(params_oid)
+//!     parameters: Some(Any::encode_from(&params_oid).unwrap())
```

Comment:
I believe this was a regression introduced in #769

Reply:
How is this a regression? The change seems like a regression.